### PR TITLE
SQ-357/speaker placeholder image

### LINKED ISSUE #357: Missing speaker image placeholder
We currently don't have a placeholder image for speakers that don't have one.

### PR DESCRIPTION
**Pull request by rock3r**
This PR adds a placeholder image for speakers without a photo. Closes #357 

image

### CODE REVIEW

File: app/src/main/java/net/squanchy/settings/view/SettingsHeaderLayout.java [hunk 1]
```diff
@@ -85,7 +85,10 @@ private void updateUserPhotoFrom(UserInfo userInfo) {
         Optional<String> photoUrl = photoUrlFor(userInfo);
         if (photoUrl.isPresent()) {
             imageLoader.load(photoUrl.get())
+                    .error(R.drawable.ic_no_avatar)
                     .into(userPhotoView);
+        } else {
```
Comment (fourlastor) [inline app/src/main/java/net/squanchy/settings/view/SettingsHeaderLayout.java:90]: that's already 💯 but maybe we can do something like `photoUrl.or(R.drawable.ic_no_avatar)` to remove the if/else? 😄 

Same applies to the other places

**Review by fourlastor — APPROVED** [review]
One question, not a blocker tho

File: app/src/main/java/net/squanchy/settings/view/SettingsHeaderLayout.java (hunk 1 above)
Comment (rock3r) [reply app/src/main/java/net/squanchy/settings/view/SettingsHeaderLayout.java:90]: Yeah maybe in the future, the problem is that those urls are strings and having an extension method on them that turns them into ints and then getting rid of type safety in the image loader to support the scenario seems... worse XD

Comment (rock3r) [reply app/src/main/java/net/squanchy/settings/view/SettingsHeaderLayout.java:90]: Actually disregard the imageloader bit; we still have to figure out a way to create a decent API for this that is actually feasible. I cannot think of a way off the top of my head

**Pull request merged**